integration_tests/general.bats

```diff
@@ -79,3 +79,15 @@ teardown_file() {
   run $IPROF --no-analysis -- bash -c "exit 55"
   [ "$status" == 55 ]
 }
+
+@test "thapi_start_stop" {
+  cc -I${THAPI_INC_DIR} ./integration_tests/thapi_start_stop.c -o thapi_start_stop \
+    -Wl,-rpath,${THAPI_LIB_DIR} -L${THAPI_LIB_DIR} -lThapi
+  $IPROF --no-analysis -- ./thapi_start_stop
+
+  start_count=`babeltrace2 $THAPI_HOME/thapi-traces | grep lttng_ust_toggle:start | wc -l`
+  assert_equal ${start_count} 1
```

Comment:
`assert_equal` doesn't seem to exist.  can just do `[  start_count == 1 ]` I suppose  : )

Reply:
Yes, it looks like it. Will fix it.